rclrs/src/error.rs

```diff
@@ -34,6 +34,8 @@ pub enum RclrsError {
         /// Index provided
         wrong_index: usize,
     },
+    /// It was attempted to add a waitable to a wait set twice.
```

Comment:
```suggestion
    /// Given waitable was already added to a wait set
```

Reply:
It was not part of this PR. It appeared there after mercedes.